### DISCUSSION TITLE
Show the Table of Contents (ToC) be default on 13.3" screens.

### DISCUSSION THREAD

**Discussion by johnnymatthews** [GitbookIO/community#232] [Feature Requests]
### What's your feature/idea?

The `@media` queries that define when to hide the Table of Contents are, in my opinion, too narrow. 13.3" is the screen size for the M1 Macbook Air; a pretty popular device.

I've attached two screenshots showing what I'm talking about. The first shows a website at 100% in the browser:

<img width="1552" alt="Screenshot 2023-06-15 at 20 52 55" src="https://github.com/GitbookIO/community/assets/9611008/b2739372-c50c-469b-a567-9d8d59c4a015">

The second shows that same page but at 90%:

<img width="1552" alt="Screenshot 2023-06-15 at 20 52 58" src="https://github.com/GitbookIO/community/assets/9611008/93a4defd-9966-4608-9646-a5eb8cc58897">

I've marked this as `high` impact. A few colleagues aren't aware of this little _hack_ and are assuming that the Gitbook service doesn't contain a table of contents.

### What's your desired outcome?

When I view a Gitbook page at 100% on a 13.3" monitor, I should be able to see the table of contents for that page.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

### Anything else?

This could be solved with custom CSS, but I saw a separate issue saying that custom CSS is not coming anytime soon. A shame, but understandable. I'm also not sure if the monitor size is the issue here, or the resolution.

**Comment by johnnymatthews** [comment]
After quickly playing around, it looks like the cutoff for the ToC is `1490` pixels. I don't know anything about Gitbook's design rules, but I reckon you could safely set it to `1400` without causing any major problems.

**Comment by dustinloflandsmith** [comment]
Thanks for posting this, @johnnymatthews! I agree wholeheartedly. This is something that came up for me recently when doing user testing on our documentation's navigation - users had a very different experience based on screen size/resolution.

Also, not to piggy-back too much on Johnny's request (and please let me know if I should open my own), but it would also be great if there were a way still have the ToC/On this page accessible when the screen is too small to show it, akin to how the page action buttons currently collapse inside of a "three dots" menu. In fact, incorporating it into that menu might be the slickest solution.

**Comment by johnnymatthews** [comment]
I received the following email from Gitbook support regarding this issue:

> This isn't possible at the moment I'm afraid - we are aware of it but it's not an issue our team can pick up at the moment.
> I appreciate this is frustrating but I would rather be honest about our planned work.

The fix for this issue should be incredibly trivial, so it's a shame that Gitbook won't be getting to it anytime soon...

**Reply by taszap** [reply]
Hey @johnnymatthews I spoke to our team and we've made some adjustments. You should be able to see the TOC on published content now, although we still have some further work to do to make this even smoother experience both in app and on published content.

**Reply by johnnymatthews** [reply]
Oh that's great! Thanks for the update @taszap!
